### DISCUSSION TITLE
能否以某种方式提供Mortal4.0的一些对局参数？

### DISCUSSION THREAD

**Discussion by Preserver6166** [Equim-chan/mjai-reviewer#134] [Ideas]
如副露率，立直率，放铳率，和牌率，自摸率，平均打点，平均铳点等统计数据？
类似于NAGA的统计信息
![FvBMy9eaMAAlIoX](https://github.com/Equim-chan/mjai-reviewer/assets/25025606/9c377d9d-4447-46df-aec9-c9f08eddc77e)

**Comment by Equim-chan** [comment]
文档里有。

https://mortal.ekyu.moe/perf/strength.html
